"PR suggestions" fails without reason for some PR

**Describe the bug**
For some PRs, the "PR suggestions" action fails without reason.
In the context of https://github.com/microsoft/vcpkg/pull/23532, this seems to be triggered by the manifest (correctly) using `version-string` instead of `version`.

**Environment**
https://github.com/microsoft/vcpkg/actions/workflows/untrustedPR.yml?query=is%3Afailure

**Failure logs** 
~~~
Succeeded in formatting the manifest files.
[detached HEAD 2da5e81] tmp
Error: Process completed with exit code 1.
~~~
https://github.com/microsoft/vcpkg/actions/runs/1990223146

--- a/.github/workflows/trustedPR.yml
+++ b/.github/workflows/trustedPR.yml
@@ -20,7 +20,7 @@ jobs:
         uses: actions/github-script@v6
         with:
           script: |
-            var artifacts = await github.actions.listWorkflowRunArtifacts({
+            var artifacts = await github.rest.actions.listWorkflowRunArtifacts({
                owner: context.repo.owner,
                repo: context.repo.repo,
                run_id: ${{github.event.workflow_run.id }},
@@ -28,7 +28,7 @@ jobs:
             var matchArtifact = artifacts.data.artifacts.filter((artifact) => {
               return artifact.name == "pr"
             })[0];
-            var download = await github.actions.downloadArtifact({
+            var download = await github.rest.actions.downloadArtifact({
                owner: context.repo.owner,
                repo: context.repo.repo,
                artifact_id: matchArtifact.id,

